internal/cmd/beta/server/start/start.go

```diff
@@ -0,0 +1,115 @@
+package start
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/client"
+	iaasUtils "github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/utils"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/spinner"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/iaas"
+	"github.com/stackitcloud/stackit-sdk-go/services/iaas/wait"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	serverIdArg = "SERVER_ID"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	ServerId string
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "start",
+		Short: "Start an existing server or allocates the server if deallocated",
+		Long:  "Start an existing server or allocates the server if deallocated.",
```

Comment:
```suggestion
		Short: "Starts an existing server or allocates the server if deallocated",
		Long:  "Starts an existing server or allocates the server if deallocated.",
```

Reply:
Same for all other commands in the PR (to match the style of all descriptions in the CLI)